svg.js.d.ts

```diff
@@ -1,3 +1,8 @@
+export = svgjs;
+export as namespace svgjs;
+
+declare var svgjs: svgjs.Library;
+

```

Comment:
i did some testing with importing the library and it looks like
```typescript
import SVG from 'svg.js'
```
and
```typescript
import * as SVG from 'svg.js'
```
returns the default `SVG()` function.
Its also possible to do 
```typescript
import {Element} from 'svg.js'
```
with that in mind I tested the old typings file and the one here and this is what i got.

**The original**
this worked fine
```typescript
import SVG from 'svg.js'
var draw: svgjs.Doc = new SVG.Doc();
```
this dose not, because the original typings only exports `svgjs.Library`
```typescript
import {Doc} from 'svg.js'
var draw: svgjs.Doc = new Doc();
```
this dose not work.
it makes SVG a object with the `default` property set to the default `SVG()` function
```typescript
import * as SVG from 'svg.js'
var draw: svgjs.Doc = new SVG.Doc();
```

**With your version**
this dose not work, it says that there is no default export
```typescript
import SVG from 'svg.js'
var draw: svgjs.Doc = new SVG.Doc();
```
but this works fine.
except for the `svgjs.Doc`, it says that `Namespace 'svgjs' has no exported member 'Doc'`
```typescript
import {Doc} from 'svg.js'
var draw: svgjs.Doc = new Doc();
```
also with your version this works.
but has the same problem with the svgjs namespace
```typescript
import * as SVG from 'svg.js'
var draw: svgjs.Doc = new Doc();
```

**Conclusion**
it looks like your version works a little better then the original.
we would just have to fix the default export and the namespace error.